app/code/Magento/CheckoutAgreements/composer.json

```
@@ -10,7 +10,8 @@
         "magento/module-backend": "*",
         "magento/module-checkout": "*",
         "magento/module-quote": "*",
-        "magento/module-store": "*"
+        "magento/module-store": "*",
+        "magento/module-cms": "*",
```

I think that last comma could lead to issues and/or warnings
```suggestion
        "magento/module-cms": "*"
```